packages/wonder-stuff-core/src/index.js

```diff
@@ -1,3 +1,16 @@
+/* eslint-disable no-console */
 // @flow
-// eslint-disable-next-line no-console
-console.log("Hello World!");
+switch (process.env.NODE_ENV) {
+    default:
+    case "production":
+        console.log("Hello production World!");
+        break;
+
+    case "development":
+        console.log("Hello development World!");
+        break;
+
+    case "test":
+        console.log("Hello test World!");
+        break;
+}
```

Comment:
Allows me to test out the `process.env.NODE_ENV` support and see how it impacts the output.